Makefile

```diff
@@ -21,3 +24,6 @@ test:
 
 clean:
 	rm -f $(TARGET)
+
+install:
+	install -D $(TARGET) $(LIBEXECDIR)/kata-containers/$(TARGET)
```

Comment:
`mkdir` first?

Reply:
@sboeuf - I have the same comment for this change as https://github.com/kata-containers/shim/pull/24#discussion_r161496203.

Reply:
And I have the same comment https://github.com/kata-containers/shim/pull/24#discussion_r161860627 :smile: